### PR TITLE
Adds breadcrumb loader to links

### PR DESCRIPTION
**Pull request by broskoski**
spa-load

There are a lot of changes from both types being regenerated and new files being converted to ts.

But the basic idea is, for whatever object we want to link to: 
- fetch the required segments
- pass those segments as state through the router
- loading page picks up the state and displays the breadcrumbs if available.

### CODE REVIEW

File: src/v2/util/getBreadcrumbPath.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { SerializeMeQueryHook_serializedMe } from '__generated__/SerializeMeQueryHook'
+import { LoadingBreadcrumbChannel } from '__generated__/LoadingBreadcrumbChannel'
+import { LoadingBreadcrumbUser } from '__generated__/LoadingBreadcrumbUser'
+import { LoadingBreadcrumbGroup } from '__generated__/LoadingBreadcrumbGroup'
+
+type Crumbable =
```
Comment (dzucconi) [inline src/v2/util/getBreadcrumbPath.ts:6]: lol this name

Comment (dzucconi) [reply src/v2/util/getBreadcrumbPath.ts:6]: vaguely disappointed it wasn't named `type Crumby`

Comment (broskoski) [reply src/v2/util/getBreadcrumbPath.ts:6]: I can do that for you

**Comment by dzucconi** [comment]
Looks good, pleased about this change which I think will make a big difference in perception.

**Pull request merged**